solr/core/src/java/org/apache/solr/schema/ManagedIndexSchemaFactory.java

```diff
@@ -148,12 +147,11 @@ public String lookupZKManagedSchemaPath() {
    */
   public Path lookupLocalManagedSchemaPath() {
     final Path legacyManagedSchemaPath =
-        Paths.get(
```

Comment:
We should try to remove usages of Paths.get generally, I remember it having some wonky behaviour in the past. Happy to see this coming in.

Reply:
Absolutely; same with Path.of.  _Ultimately_ we should ForbiddenApi check this so that it's rare to call this except for a few authorized places.